xonsh/ptk/completer.py

```diff
@@ -46,8 +46,13 @@ def reserve_space(self):
             #old layout to be removed at next ptk release
             window = cli.application.layout.children[1].children[1].content
         except AttributeError:
-            #new layout to become default
-            window = cli.application.layout.children[1].content
+            try:
+                # This is the layout for ptk 1.0
+                window = cli.application.layout.children[0].content.children[1]
```

Comment:
Rather than doing a try-except, could you please do an `if ptk version >= 1` here? The reason for the other try except block is because PTK didn't used to report its version numbers :)